src/intTest/org/onflow/flow/sdk/ExposeAccountKeyIssueTest.kt

```diff
@@ -128,18 +136,26 @@ class ExposeAccountKeyIssueTest {
                 arg { ufix64(1000) }
             }
         }.sendAndWaitForSeal()
-            .throwOnError()
 
-        account = requireNotNull(flow.getAccountAtLatestBlock(newAccountAddress))
-        assertEquals(2, account.keys.size)
-        assertEquals(pair1.public.hex, account.keys[0].publicKey.base16Value)
-        assertEquals(pair2.public.hex, account.keys[1].publicKey.base16Value)
-        assertFalse(account.keys[0].revoked)
-        assertFalse(account.keys[1].revoked)
+        val addKeyEvent = when (addKeyResult) {
+            is FlowAccessApi.FlowResult.Success -> addKeyResult.data
+            is FlowAccessApi.FlowResult.Error -> throw IllegalStateException("Failed to add key: ${addKeyResult.message}", addKeyResult.throwable)
+        }
+
+        val updatedAccount = when (val updatedAccountResult = flow.getAccountAtLatestBlock(newAccountAddress)) {
+            is FlowAccessApi.FlowResult.Success -> updatedAccountResult.data
+            is FlowAccessApi.FlowResult.Error -> throw IllegalStateException("Failed to get updated account at latest block: ${updatedAccountResult.message}", updatedAccountResult.throwable)
+        }
```

Comment:
It would be nice if the boilerplate in these repeated blocks could be served by a utility method to reduce the repetition, perhaps in IntegrationTestUtils? Wonder if the Coderabbit AI bot can suggest a refactoring that would work

Reply:
Agree, just pushed some updates to the integration tests which refactor the response structure to remove the redundant code throughout. Will go through the other test files as well to adjust where possible. Thanks